dev/TabView/TabViewItemAutomationPeer.cpp

```diff
@@ -24,3 +24,25 @@ winrt::AutomationControlType TabViewItemAutomationPeer::GetAutomationControlType
     return winrt::AutomationControlType::TabItem;
 }
 
+
+winrt::hstring TabViewItemAutomationPeer::GetNameCore()
+{
+    winrt::hstring returnHString = __super::GetNameCore();
+
+    // If a name hasn't been provided by AutomationProperties.Name in markup:
+    if (returnHString.empty())
+    {
+        if (auto tvi = Owner().try_as<winrt::TabViewItem>())
+        {
+            if (auto content = tvi.Header())
+            {
+                if (auto stringableName = content.try_as<winrt::IStringable>())
```

Comment:
I remember I fixed a TreeView bug recently where the string content in cx app was passed in as hstring directly and didn't implement IStringable interface. Maybe do some tests in cx app and see if it works there?

Reply:
The TreeView bug fix, https://github.com/microsoft/microsoft-ui-xaml/pull/1357.

Reply:
Thanks. That's helpful. I'll have a look.